Documentation/user-guides/cluster-monitoring.md

```diff
@@ -361,6 +361,32 @@ spec:
     interval: 15s
   - port: http-metrics-skydns
     interval: 15s
+---
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: ServiceMonitor
+metadata:
+  name: prometheus
+  labels:
+    k8s-apps: prometheus
```

Comment:
Shouldn't this be `prometheus: k8s` to match other prometheus components ? I think `k8s-apps` is reserved to `kubernetes` components like the `apiserver` or the `kube-controller`.